src/src/services/models/explorer/createAppModel.ts

```diff
@@ -1251,6 +1264,7 @@ function createAppModel(appConfig: IAppInitialConfig) {
     ): void {
       const modelState: IAppModelState = model.getState();
       const sortFields = modelState?.config?.table?.sortFields;
+      console.log(rawData);
```

Comment:
debugging?

Reply:
Yes, didn't push the latest changes for removing those